### PR TITLE
Update zones.json

### PR DESCRIPTION
**Pull request by xomanuel**
Adding data for the Philippines

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -3763,6 +3763,25 @@
     "timezone": "Pacific/Port_Moresby"
   },
   "PH": {
+    "_comment": "Capacity is based on Philippine Department of Energy's Power Situation Report as of 2020 - http://www.doe.gov.ph",
```
Comment (VIKTORVAV99) [inline config/zones.json:3766]: Can you move this into `DATA_SOURCES.md` with a full link to the report instead?

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
Hi and welcome!

At first glance this looks good, just one small change is required so it matches the rest of the sources but that's it.

Thanks for opening this PR! 🎉

File: config/zones.json (hunk 1 above)
Comment (xomanuel) [reply config/zones.json:3766]: Thank you for considering my input. Included the source in DATA_SOURCES.md

**Review by VIKTORVAV99 — APPROVED** [review]
Saw the other PR, let's get this merged!

**Pull request merged**